miqcli/collections/__init__.py

```diff
@@ -24,8 +24,6 @@ class Collection(ClientAPI):
     Main option is to save the settings
```

Comment:
Now that the collection class main purpose is not to save the configuration settings, could we remove this statement from the class doc string?

Reply:
agreed